lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -85,8 +93,8 @@ Object {
   "observedFirstVisualChange": 1105,
   "observedFirstVisualChangeTs": 713038128064,
   "observedLargestContentfulPaint": 1122,
-  "observedLargestContentfulPaintAllFrames": undefined,
-  "observedLargestContentfulPaintAllFramesTs": undefined,
+  "observedLargestContentfulPaintAllFrames": 1122,
```

Comment:
hm, why did this one change? I wouldn't have expected movement here.

Reply:
Previously, if there were no `FrameTreeCommittedInBrowser` events, only FCP-AF would fall back to FCP because we needed to avoid a `NO_FCP` error. LCP-AF didn't need to fall back because a `NO_LCP_ALL_FRAMES` error didn't brick a bunch of existing tests.

Now that `frameTreeEvents` falls back to `frameEvents`, LCP-AF falls back to LCP because it looks at `frameTreeEvents`.